src/test/java/de/tum/in/www1/artemis/metis/linkpreview/LinkPreviewIntegrationTest.java

```diff
@@ -32,6 +39,12 @@ class LinkPreviewIntegrationTest extends AbstractSpringIntegrationIndependentTes
     @Autowired
     private CacheManager cacheManager;
 
+    @Mock
```

Comment:
Why do they need to be Mocks? If I'm not mistaken they are simply returned as mocked response., so they can be just regular POJO without Mockito annotations

Reply:
The OpenGraph has a private constructor therefore cannot be instantiated in the test. That is why I used @Mock.